app/web/features/NotFoundPage.tsx

```diff
@@ -23,6 +25,11 @@ const useStyles = makeStyles((theme) => ({
 export default function NotFoundPage() {
   const { t } = useTranslation(GLOBAL);
   const classes = useStyles();
+  const {
+    authState: { authenticated },
+  } = useAuthContext();
+  const [isMounted, setIsMounted] = useState(false);
+  useEffect(() => setIsMounted(true), []);
```

Comment:
There's an `useIsMounted` hook in `utils/hooks` we can use instead?

Reply:
It doesn't work in this case since it uses a ref and not state, so I think the component wasn't re-rendering

Reply:
Does it need to re-render though if at the logic bit, it's already reading the up-to-date value from `.current` for `mounted`? And I believe the `authState` stuff is already in state so that should be fine?

Reply:
It always showed the `baseRoute` when I tried using the existing hook. I assumed it was because it would render and `isMounted` would be false, it would then update to true but not render again because it's a ref.

Reply:
Ahh fair enough, would be interesting to see if we encounter this enough and whether it's worth updating the original hook in the future